api/tests/test_utils.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+from api.utils import filter_valid_destinations, validate_send_message_payload
+
+REGION_FI_VALID_PHONE_NUMBERS = [
```

Comment:
The numbers and input formats that we know that should be valid and we should support, should be added here. You can actually also use alphabets as numbers with phonenumbers-library, which is a bit strange feature, but before this PR, all the inputs were valid, so this does not make it worse at least.

Reply:
Should we use the `format_number` only when the phone number does not include any letters?

Reply:
The Quriiri API encourages to use the international formats (but we have not used them earlier, so this is already kind of improvement) https://docs.quriiri.fi/docs/quriiri/send-sms/operations/create-a

Reply:
Filtering those alphabetic numbers out sounds like a good idea to me, at least I don't know of any such numbers in Finland that would have a legitimate use case, do you?

Reply:
1. We could filter them out on our end
2. We can just send them and let Quriiri do that decision

Currently, the production code is sending those to Quriiri. This PR does now always use the international formatter, so the alphabets will be replaced with numbers.

I think the best at this point is to trust in Google's phone number lilbrary, but I'll add a checker, that if the letters are included, we don't use the Google's formatter.